easybuild/easyconfigs/p/PyClone/PyClone-2020.9b2-GCCcore-10.2.0-Python-3.8.6.eb

```diff
@@ -0,0 +1,48 @@
+# easybuild easyconfig
+#
+#  John Dey <jfdey@fredhtuch.org>
+#
+#  Fred Hutchinson Cancer Research Center - Seattle Washington - US
+#
+easyblock = 'Bundle'
+
+name = 'PyClone'
+version = '2020.9b2'
+versionsuffix = '-Python-%(pyver)s'
```

Comment:
@fizwit We've dropped this versionsuffix for the `2020b` toolchains onwards when using Python 3 (we'll still use it in case Python 2 is really needed as a runtime dep, by means of exception)

Reply:
good to know thanks,